Shared/UI/Views/PersonView.swift

```diff
@@ -13,27 +13,6 @@ class PersonView: DisplayableImageView {
     
     // MARK: - Properties
 
-    var initials: String? {
-        didSet {
-            self.setImageFrom(initials: self.initials)
-        }
-    }
-
-    private let label = ThemeLabel(font: .regularBold)
-
-    private func setImageFrom(initials: String?) {
-        guard let initials = initials else {
-            self.label.text = nil
-            return
-        }
-
-        self.label.isHidden = false
-        self.label.setText(initials.uppercased())
-        self.label.textAlignment = .center
-        self.state = .success
-        self.layoutNow()
-    }
-
```

Comment:
Unneeded now because we programmatically generate a name initials image for contacts.